app/assets/stylesheets/application.css

```diff
@@ -33,4 +33,5 @@
  *= require lara-typescript
  *= require styled-admin-forms
  *= require project-editing
+ *= require shared/_loading
```

Comment:
For this and the related addition of app/assets/stylesheets/shared/_loading.scss, see comment below on app/views/shared/_loading.html.haml.